style.css

```diff
@@ -0,0 +1,256 @@
+* {
+  margin: 0;
+  padding: 0;
+  scrollbar-width: none;
+}
+
+body {
+  background-image: url(img/background.jpg);
+  background-repeat: no-repeat;
+  background-size: 420% auto;
+  background-position-x: 55%;
+  height: 100vh;
+  display: grid;
+  grid-template-rows: auto 70px;
+  grid-gap: 20px;
+}
+
+/* header */
+
+header {
+  display: block;
+  padding: 10px 15px 0px;
+}
+
+header h2 {
+  margin: 10px;
+  margin-top: 80px;
+  font-family: "Rubik", sans-serif;
+  font-weight: 400;
+  font-size: 1.7em;
+  color: aliceblue;
+  text-shadow: 0 0 2px black;
+}
+
+#date {
+  margin: 10px;
+  font-family: "Rubik", sans-serif;
+  font-weight: 400;
+  font-size: 0.9em;
+  color: aliceblue;
+  text-shadow: 0 0 2px black;
+}
+
+/* /header */
+
+/* homeScreen */
+
+#homeScreen {
+  display: flex;
+  flex-direction: column;
+  transition: all 0.2s;
+}
+
+/* homeScreen - taskToday */
+
+#homeScreen main {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  flex-grow: 1;
+}
+
+#homeScreen main #toToday {
+  margin: 5px auto 5px;
+  width: 325px;
+  height: fit-content;
+  overflow: hidden;
+  padding: 15px;
+  background-color: rgba(127, 127, 127, 0.3);
+  backdrop-filter: blur(6px);
+  border-radius: 10px;
+  box-shadow: 0 0 1px black;
+}
+
+#homeScreen main #toToday .heading {
+  font-size: 0.9em;
+  color: rgb(0, 94, 165);
+  margin: 10px;
+  left: 10px;
+}
+
+#homeScreen main #toToday ul {
+  max-height: 300px;
+  list-style-position: inside;
+  overflow-y: scroll;
+  width: 100%;
+  list-style-type: circle;
+  font-family: "LXGW WenKai TC", cursive;
+  font-weight: 400;
+  font-style: normal;
+}
+
+#homeScreen main #toToday ul li {
+  color: rgb(79, 79, 79);
+  user-select: none;
+  padding: 5px;
+  transition: background-color 0.2s;
+  border-radius: 5px;
+}
+
+#homeScreen main #toToday ul .complete {
+  text-decoration: line-through;
+}
+
+#homeScreen main #toToday ul li:hover {
+  background-color: rgba(0, 0, 0, 0.1);
+}
+
+/* homeScreen - otherTask */
+
+#homeScreen main #toDo h3 {
+  color: white;
+  margin-left: 10px;
+  text-shadow: 0 0 2ps black;
+}
+
+#homeScreen main #toDo hr {
+  color: white;
+  width: 50%;
+  margin: 10px;
+}
+
+#homeScreen main #toDo #toDoList {
+  max-height: 150px;
+  overflow-y: scroll;
+}
+
+#homeScreen main #toDo #toDoList ul .task {
+  background-color: rgba(179, 179, 179, 0.3);
+  backdrop-filter: blur(6px);
+  border-radius: 10px;
+  margin: 5px auto;
+  padding: 15px 10px;
+  width: 350px;
```

Comment:
auch hier würde ich eher versuchen die Größen der Elemente intrinsisch, also basierend auf ihrem Inhalt, Schriftgröße und Innenabständen, entstehen zu lassen anstatt sie von Außen zu forcieren.

Das führt in der Regel zu Layouts die besser mit unterschiedlichen Viewports umgehen können.